web/sites/default/config/moderation_dashboard.settings.yml

```diff
@@ -1,3 +1,3 @@
-redirect_on_login: false
```

Comment:
This is supposed to be false. I think you may have accidentally overwritten some config from master in one of your commits.